pkg/controller/volume/attachdetach/populator/desired_state_of_world_populator.go

```diff
@@ -113,3 +126,21 @@ func (dswp *desiredStateOfWorldPopulator) findAndRemoveDeletedPods() {
 		dswp.desiredStateOfWorld.DeletePod(dswPodUID, dswPodToAdd.VolumeName, dswPodToAdd.NodeName)
 	}
 }
+
+func (dswp *desiredStateOfWorldPopulator) findAndAddActivePods() {
+	pods, err := dswp.podLister.List(labels.Everything())
```

Comment:
It's possible that fetching and processing all pods from the informer cache once per minute could result in perf issues related to memory. When the controller starts (or restarts), it takes that kind of hit already.

I say let's get it merged, give it plenty of time to bake for 1.7, and if the scale tests freak out we can consider things like reducing the frequency of this. CC @kubernetes/sig-scalability-misc